runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -1035,6 +1048,22 @@ void chpl_comm_barrier(const char *msg) {
 
 void chpl_comm_pre_task_exit(int all) {
   if (all) {
+
+    if (chpl_nodeID == 0) {
+     int node;
+     for (node = 0; node < chpl_numNodes; node++) {
+       if (node != chpl_nodeID) {
```

Comment:
Just noticed that I could just start at 1 instead of 0 and skip this conditional -- thoughts?

Reply:
I saw this in the original branch and thought about mentioning it, but then I considered that when we expand on the capability you're adding here so that any node (rather than just node 0) can initiate an orderly shutdown we'll probably end up with code just like this.  That being so, I'm fine with leaving it how it is.